store/tikv/split_region.go

```diff
@@ -29,12 +28,7 @@ import (
 
 // SplitRegion splits the region contains splitKey into 2 regions: [start,
 // splitKey) and [splitKey, end).
-func (s *tikvStore) SplitRegion(splitKey kv.Key) error {
-	_, err := s.splitRegion(splitKey)
-	return err
-}
-
-func (s *tikvStore) splitRegion(splitKey kv.Key) (*metapb.Region, error) {
+func (s *tikvStore) SplitRegion(splitKey kv.Key, scatter bool) (uint64, error) {
```

Comment:
Need comment for the returned `uint64` value.

Reply:
I think `SplitRegion` is better just used for splitting region, scatter region is another action. Use a boolean to indicate different actions is not clean code. Keep `SplitRegionAndScatter` is better?

Reply:
`SplitRegionAndScatter` is even bad. I think it’s worse to combine the names of the actions together as the function name. And we also need an interface named `SplitRegion`. The best way is to add a standalone interface for scattering, but now we have at least one interface saved, and split and scatter are really strongly related. If we need to use scatter separately in the future, I think we can separate them again.